LICENSE.txt

```diff
@@ -1,7 +1,7 @@
 
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```

Comment:
IANAL - but I really don't think we should be changing the text of the license in any way here.  Maybe break other downstream tooling for all we know.  Flag a license change and review of OkHttp usage in some companies?